packages/materialized/package.json

```diff
@@ -12,8 +12,9 @@
   "main": "dist/ts-build-materialized/index.js",
   "typings": "dist/ts-build-materialized/index.d.ts",
   "scripts": {
-    "bundle": "../../node_modules/.bin/webpack --config webpack/webpack.build.js --env=production --display-error-details",
+    "bundle": "../../node_modules/.bin/webpack --config ../../webpack.build.js --env=production --display-error-details",
```

Comment:
devDeps for this package.json seem to be missing
* copy-webpack-plugin
* source-map-loader
* ts-loader
* webpack-dev-server